convert2rhel/logger.py

```diff
@@ -82,7 +82,9 @@ class LogLevelTask(object):
 
 class LogLevelFile(object):
     level = 5
-    label = "FILE"
+    # Label messages DEBUG as it is contains the same messages as debug, just that they always go
+    # to the log file.
+    label = "DEBUG"
```

Comment:
Docs at the top of the file needs to change. Seems to be the only thing left

Reply:
nice catch, indeed.

Reply:
Done.